.theo/formats/stories.mdx.js

```diff
@@ -54,12 +79,15 @@ function mdxProp(prop) {
  * @returns {String}
  */
 function mdxProps(props) {
-  const rows = props.map(mdxProp);
+  const commentedProps = props.filter((prop) => !!prop.comment);
```

Comment:
Nit. I doubt this matters, but `'comment' in props` would return `true` for props with an empty comment, while this code wouldn't. But what you have is probably the behavior you want.

Reply:
Yeah, I don't want it to return true for an empty comment.

Is this blocking approval?

Reply:
No, approved. Just trying to be thorough.

Reply:
Can confirm this is working as intended — to filter the props array and return only props with comments.